### PR TITLE
Drop ID from parameter

### PR DESCRIPTION
**Pull request by ad-m**
Drop ID from parameters at all. In addiction update description where I see that.

### CODE REVIEW

File: bin/generic/rename.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = resource => Cli.createCommand('rename', {
   , options: Object.assign({}, options, resource.options)
   , handler: args => args.helpers.api
         .patch(`${args.$node.parent.config.url(args)}/${args.id}`, {
-            name: args.newname
+            name: args["new-name"]
```
Comment (fredericosilva) [inline bin/generic/rename.js]: Avoid use double quotes, use single one

File: bin/_plugins/outputFormat.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const queryFilter = (args, result) => {
         result = eval(args.transform)(result);
     }
 
-    const query = args.query || config.defaultQuery || "[].['_id']";
+    const query = args.query || config.defaultQuery || '[].[_id]';
```
Comment (ad-m) [inline bin/_plugins/outputFormat.js]: Fix quotes style & fix bug:
```
$ h1 vm list --query "[].['_id']"
0
_id
$ h1 vm list --query "[].[_id]"
0
5b0e9c19e8f71366b5d744ec
```

File: bin/generic/rename.js (hunk 1 above)
Comment (ad-m) [reply bin/generic/rename.js]: Fixed & added tests for that.

File: bin/_plugins/outputFormat.js (hunk 2 above)
Comment (fredericosilva) [reply bin/_plugins/outputFormat.js]: Does it accepts '[].["_id"]' ?

Comment (ad-m) [reply bin/_plugins/outputFormat.js]: @fredericosilva , yes.
```
$ h1 vm list --query '[].["_id"]'
0
5b155c1e310321f9a2c0b070
```

**Comment by ad-m** [comment]
Fixed. Ready to squash & merge.

**Pull request merged**